index.html

```diff
@@ -466,10 +466,10 @@
       automatically generated by the application server or an optimization
       proxy (e.g. a CDN).</p>
       <pre class="example">
-  Link: &lt;https://example.com/font.woff&gt;; rel=preload; as=font
+  Link: &lt;https://example.com/font.woff2&gt;; rel=preload; as=font; type='font/woff2'
   Link: &lt;https://example.com/app/script.js&gt;; rel=preload; as=script
   Link: &lt;https://example.com/logo-hires.jpg&gt;; rel=preload; as=image
-  Link: &lt;https://fonts.example.com/font.woff&gt;; rel=preload; as=font; crossorigin
+  Link: &lt;https://fonts.example.com/font.woff2&gt;; rel=preload; as=font; crossorigin; type='font/woff2'
```

Comment:
Which type value are you using? Can you file a crbug.com with a test case and send it my way?